frontend/jammy/handle_deb.go

```diff
@@ -145,26 +144,59 @@ func customRepoMounts(worker llb.State, repos []dalec.PackageRepositoryConfig, s
 	return dalec.WithRunOptions(withRepos, withData, keyMounts), nil
 }
 
-func installPackages(ls []string, rOpts ...llb.RunOption) llb.RunOption {
+func installPackages(ls []string, opts ...llb.ConstraintsOpt) llb.RunOption {
+	script := llb.Scratch().File(
+		llb.Mkfile("install.sh", 0o755, []byte(`#!/usr/bin/env sh
+set -ex
+
+# Make sure any cached data from local repos is purged since this should not
+# be shared between builds.
+rm -f /var/lib/apt/lists/_*
```

Comment:
cached local repo data is always under a dir prefixed by `_`?

Reply:
Based on what I've seen, yes.